src/sidebar/services/root-thread.js

```diff
@@ -85,15 +92,7 @@ export default function RootThread(
 
     // Get the currently loaded annotations and the set of inputs which
     // determines what is visible and build the visible thread structure
-    return buildThread(state.annotations.annotations, {
-      forceVisible: truthyKeys(state.selection.forceVisible),
-      expanded: store.expandedThreads(),
-      highlighted: state.selection.highlighted,
-      selected: truthyKeys(store.getSelectedAnnotationMap() || {}),
-      sortCompareFn: sortFn,
-      filterFn: filterFn,
-      threadFilterFn: threadFilterFn,
-    });
+    return buildThread(state.annotations.annotations, options);
```

Comment:
Previously `filterFn` and `threadFilterFn` could be undefined, which doesn't match the JSDoc for `options` which says that `filterFn` is an optional property that, if set, should be a function. This change restructures this code so we don't set these optional properties at all if there is no filter.